IdentityCore/src/util/MSIDCryptoHelper.h

```diff
@@ -27,8 +27,8 @@
 
 #import <Foundation/Foundation.h>
 
-@interface MSIDWebviewRequest : NSObject
+@interface MSIDCryptoHelper : NSObject
 
-@property NSURL *startURL;
++ (NSData *)msidSHA256fromString:(NSString *)string;
```

Comment:
Don't we already have this functionality in some category (e.g. msidComputeSHA256)?